packages/react/src/components/Setup/ProfileSelectionModal.tsx

```diff
@@ -111,6 +116,24 @@ const ProfileSelectionModalInternal: React.FC<Props> = ({ onAddProfile, onSetMod
               value={getValues().secretAccessKey}
             />
           </FormField>
+          <FormField
+            id="session-token"
+            label={
+              <span>
+                AWS session token <i>- optional</i>
+              </span>
+            }
+          >
+            <Controller
+              as={Textarea}
+              name="sessionToken"
+              rows={5}
+              control={control}
+              onChange={([e]) => e.detail.value}
+              placeholder="Please enter AWS session token"
```

Comment:
Should this somehow convey that session is optional/only for short term creds?

Reply:
Yeah, we mention that it's optional in a different element of this model. Line 123.